client/components/seo/preview-upgrade-nudge/index.jsx

```diff
@@ -50,17 +66,45 @@ const businessPlanFeatures = [
 	FEATURE_NO_BRANDING
 ];
 
-const SeoPreviewNudge = ( { translate, domain, plan = {}, businessPlan = {} } ) => {
-	const planPrice = isFreePlan( plan )
-		? translate( 'Free for life' )
-		: translate( '%(price)s per month, billed yearly', {
-			args: {
-				price: formatCurrency( plan.raw_price / 12, plan.currency_code )
-			}
-		} );
-	const businessPlanPrice = formatCurrency( businessPlan.raw_price / 12, businessPlan.currency_code );
-	const featuresToShow = businessPlanFeatures.filter( feature => ! planHasFeature( plan.product_slug, feature ) );
+const jetpackBusinessPlanFeatures = compact( [
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME,
+	FEATURE_BACKUP_ARCHIVE_UNLIMITED,
+	FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED,
+	FEATURE_AUTOMATED_RESTORES,
+	FEATURE_SPAM_AKISMET_PLUS,
+	FEATURE_EASY_SITE_MIGRATION,
+	FEATURE_PREMIUM_SUPPORT,
+	FEATURE_WORDADS_INSTANT,
+	FEATURE_VIDEO_UPLOADS_JETPACK_PRO,
+	FEATURE_MALWARE_SCANNING_DAILY_AND_ON_DEMAND,
+	FEATURE_ONE_CLICK_THREAT_RESOLUTION,
+	FEATURE_GOOGLE_ANALYTICS,
+	isEnabled( 'republicize' ) && FEATURE_REPUBLICIZE,
+	isEnabled( 'publicize-scheduling' ) && FEATURE_REPUBLICIZE_SCHEDULING
+] );
```

Comment:
It might be simpler to do this as `getPlan( PLAN_JETPACK_BUSINESS ).getFeatures()`.